src/components/Profile/index.js

```diff
@@ -8,37 +8,37 @@ import Button from '../Button/Button';
 import {
   userSelectors, walletSelectors, blockchainSelectors, identitySelectors, onboardingSelectors,
 } from '../../redux/selectors';
-import { formatDollars, formatMerits, parseDollars } from '../../utils/walletHelpers';
-
+import { formatDollars, formatMerits } from '../../utils/walletHelpers'; // parseDollars
+import { ReactComponent as GlobeIcon } from '../../assets/icons/globe.svg';
 import truncate from '../../utils/truncate';
 
 import styles from './styles.module.scss';
 
-import eCardImage from '../../assets/images/e_card_image.svg';
+// import eCardImage from '../../assets/images/e_card_image.svg';
 import liberlandEmblemImage from '../../assets/images/liberlandEmblem.svg';
-import locationImage from '../../assets/icons/location.svg';
-import languagesImage from '../../assets/icons/languages.svg';
-import occupationImage from '../../assets/icons/occuoation.svg';
-import genderImage from '../../assets/icons/gender.svg';
-import startOfKyc from '../../assets/icons/startOfKyc.svg';
+// import locationImage from '../../assets/icons/location.svg';
+// import languagesImage from '../../assets/icons/languages.svg';
+// import occupationImage from '../../assets/icons/occuoation.svg';
+// import genderImage from '../../assets/icons/gender.svg';
+// import startOfKyc from '../../assets/icons/startOfKyc.svg';
 import Card from '../Card';
 import { OnchainIdentityModal } from '../Modals';
-import { congressActions, identityActions, onBoardingActions } from '../../redux/actions';
+import { identityActions, onBoardingActions } from '../../redux/actions'; // congressActions
 import {
   parseLegal, parseIdentityData, parseDOB, parseAdditionalFlag, parseCitizenshipJudgement,
 } from '../../utils/identityParser';
-import { getComplimentaryLLD, maybeGetApprovedEresidency } from '../../api/backend';
+// import { getComplimentaryLLD, maybeGetApprovedEresidency } from '../../api/backend';
 
 function Profile({ className }) {
   const userName = useSelector(userSelectors.selectUserGivenName);
   const lastName = useSelector(userSelectors.selectUserFamilyName);
   const walletAddress = useSelector(blockchainSelectors.userWalletAddressSelector);
-  const userRole = useSelector(userSelectors.selectUserRole);
-  const aboutUser = useSelector(userSelectors.selectUserAbout);
-  const originFrom = useSelector(userSelectors.selectUserOrigin);
-  const language = useSelector(userSelectors.selectUserLanguages);
-  const occupation = useSelector(userSelectors.selectUserOccupation);
-  const gender = useSelector(userSelectors.selectUserGender);
+  // const userRole = useSelector(userSelectors.selectUserRole);
+  // const aboutUser = useSelector(userSelectors.selectUserAbout);
+  // const originFrom = useSelector(userSelectors.selectUserOrigin);
```

Comment:
If not used anywhere, delete references and selectors, but check that they are really not used.